README.md

```diff
@@ -5,40 +5,29 @@ It also contains scripts for local development of Mobile Services (using `Minish
 
 ### Prerequisites:
 * Ansible 2.7.6
```

Comment:
Should we update this version of ansible - current version is 2.8.5.

Reply:
well, this version still works, so unless this version is not supported, we can keep it here. I will add `and above` to it instead to make it clear.